### PR TITLE
docs: add changeover procedure

### PR DESCRIPTION
**Pull request by MSalopek**
## Description

Closes: #1069 

Add docs for changeover procedure.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary 
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

### CODE REVIEW

File: docs/docs/consumer-development/changeover-procedure.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+---
+sidebar_position: 5
+---
+
+# Changeover Procedure
+
+Chains that were not initially launched as consumers of replicated security can still participate in the protocol and leverage the economic security of the provider chain. The process where a sovereign chain transitions to being a replicated consumer chain is called the **changeover procedure** and is part of the interchain security protocol. After the changeover, the new consumer chain will retain all existing state, including the IBC clients, connections and channels already established by the chain.
```
Comment (smarshall-spitzbart) [inline docs/docs/consumer-development/changeover-procedure.md:7]: Seems "sovereign to consumer" has stuck, but we've committed to something else in the code. See https://github.com/cosmos/interchain-security/pull/757#discussion_r1139884029

File: docs/docs/consumer-development/changeover-procedure.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+The relevant protocol specification is available [here](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#channel-initialization-existing-chains).
+
+## Overview
+
+Sovereign to consumer changeover procedure can rougly be separated into 4 parts:
+
+### 1. ConsumerAddition proposal submitted to the `provider` chain
+The proposal is equivalen to the "normal" ConsumerAddition proposal submitted by new consumer chains.
```
Comment (smarshall-spitzbart) [inline docs/docs/consumer-development/changeover-procedure.md:16]:
```suggestion
The proposal is equivalent to the "normal" ConsumerAddition proposal submitted by new consumer chains.
```

File: docs/docs/consumer-development/changeover-procedure.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+The upgrade height in the proposal should correspond to a height that is after the `spawn_time` in the consumer addition proposal submitted to the `provider` chain.
+:::
+
+Otherwise, the upgrade is indistinguishable from a regular on-chain upgrade proposal.
+
+### 3. spawn time is reached
+
+When the `spawn_time` is reached on the `provider` it will generate a `ConsumerGenesis` that contains the validator set that will supercede the `sovereign` validator set.
+
+This `ConsumerGenesis` must be available on the sovereign chain during the on-chain upgrade.
```
Comment (smarshall-spitzbart) [inline docs/docs/consumer-development/changeover-procedure.md:87]: Should we add in a note about having to manually mutate the JSON to set PreCCV = true:?

File: docs/docs/consumer-development/changeover-procedure.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+### 4. sovereign chain upgrade
+
+Performing the on-chain upgrade on the sovereign chain will add the `ccv/consumer` module and allow the chain to become a `consumer` of replicated security.
+
+:::caution
+The `ConsumerGenesis` must be exported to a file called `genesis.json` and placed in the following folder on the sovereign chain before the upgade: `<CURRENT_USER_DIR>/.sovereign/config/genesis.json` (`<CURRENT_USER>` is the home directory of the user executing the chain process)
```
Comment (smarshall-spitzbart) [inline docs/docs/consumer-development/changeover-procedure.md:94]: I think this is only true of Stride and the dummy app, but overall the JSON doesn't have to be configured this way. Maybe I'm wrong tho

File: docs/docs/introduction/terminology.md [hunk 5]
```diff
@@ -21,3 +21,16 @@ A particular protocol/implementation of Interchain Security that fully replicate
 ## Mesh security
 
 A protocol built on IBC that allows delegators on a cosmos chain to re-delegate their stake to validators in another chain's own validator set, using the original chain's token (which remains bonded on the original chain). For a deeper exploration of mesh security, see [Replicated vs. Mesh Security on the Informal Blog](https://informal.systems/blog/replicated-vs-mesh-security).
+
+## Consumer Chain
+
+Chain that is secured by the validator set of the provider, instead of its own.
+Replicated security allows the provider chain validator set to validate blocks on the consumer chain.
+
+## Sovereign Chain
```
Comment (smarshall-spitzbart) [inline docs/docs/introduction/terminology.md:30]: Either change this to standalone or make a note that the code uses different term that means the same thing

File: docs/docs/consumer-development/changeover-procedure.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
+---
+sidebar_position: 5
+---
+
+# Changeover Procedure
+
+Chains that were not initially launched as consumers of replicated security can still participate in the protocol and leverage the economic security of the provider chain. The process where a sovereign chain transitions to being a replicated consumer chain is called the **changeover procedure** and is part of the interchain security protocol. After the changeover, the new consumer chain will retain all existing state, including the IBC clients, connections and channels already established by the chain.
+
+The relevant protocol specification is available [here](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#channel-initialization-existing-chains).
```
Comment (smarshall-spitzbart) [inline docs/docs/consumer-development/changeover-procedure.md:9]: Should prob link the ADR too right?

**Review by smarshall-spitzbart — COMMENTED** [review]
Great work! Very detailed. Just some minor comments

File: docs/docs/consumer-development/changeover-procedure.md (hunk 1 above)
Comment (MSalopek) [reply docs/docs/consumer-development/changeover-procedure.md:7]: Good catch!

File: docs/docs/consumer-development/changeover-procedure.md (hunk 3 above)
Comment (MSalopek) [reply docs/docs/consumer-development/changeover-procedure.md:87]: Good catcH!

File: docs/docs/consumer-development/changeover-procedure.md (hunk 4 above)
Comment (MSalopek) [reply docs/docs/consumer-development/changeover-procedure.md:94]: Oh yeah. We'd need to actually write instructions on how to perform the upgrade.

File: docs/docs/consumer-development/changeover-procedure.md (hunk 6 above)
Comment (MSalopek) [reply docs/docs/consumer-development/changeover-procedure.md:9]: Linked, need it to get merged first

File: docs/docs/introduction/terminology.md [hunk 7]
```diff
@@ -21,3 +21,16 @@ A particular protocol/implementation of Interchain Security that fully replicate
...
 A protocol built on IBC that allows delegators on a cosmos chain to re-delegate their stake to validators in another chain's own validator set, using the original chain's token (which remains bonded on the original chain). For a deeper exploration of mesh security, see [Replicated vs. Mesh Security on the Informal Blog](https://informal.systems/blog/replicated-vs-mesh-security).
+
+## Consumer Chain
+
+Chain that is secured by the validator set of the provider, instead of its own.
+Replicated security allows the provider chain validator set to validate blocks on the consumer chain.
+
+## Standalone Chain
+
+Chain that is secured by its own validator set. This chain does not participate in replicated security.
```
Comment (smarshall-spitzbart) [inline docs/docs/introduction/terminology.md:32]: Might be useful to add a note that "standalone chain" and "sovereign chain" may be used interchangeably external to this repo, but the former is more accurate

**Review by smarshall-spitzbart — APPROVED** [review]
Looks great thanks!

**Review by mpoke — APPROVED** [review]
Great work @MSalopek

File: docs/docs/consumer-development/changeover-procedure.md [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+When the `spawn_time` is reached on the `provider` it will generate a `ConsumerGenesis` that contains the validator set that will supercede the `standalone` validator set.
+
+This `ConsumerGenesis` must be available on the standalone chain during the on-chain upgrade.
+
+### 4. standalone chain upgrade
+
+Performing the on-chain upgrade on the standalone chain will add the `ccv/consumer` module and allow the chain to become a `consumer` of replicated security.
+
+:::caution
+The `ConsumerGenesis` must be exported to a file called `genesis.json` and placed in the following folder on the standalone chain before the upgade: `<CURRENT_USER_DIR>/.sovereign/config/genesis.json` (`<CURRENT_USER>` is the home directory of the user executing the chain process)
```
[the remark this answers is not in the document]
Comment (MSalopek) [reply docs/docs/consumer-development/changeover-procedure.md]: Changed entire section to say that the exact file and folder are dictated by the upgrade doce on the `standalone` chain.

**Pull request merged**